frontend/src/components/OrderPage.js

```diff
@@ -113,6 +122,44 @@ class OrderPage extends Component {
     this.getOrderDetails(this.state.orderId);
   }
 
+  handleWebln = (data) => {
+    getWebln()
+      .then((webln) => {
+        // If Webln implements locked payments compatibility, this logic won't be necesary anymore
+        if (data.is_maker & data.status == 0) {
+          webln.sendPayment(data.bond_invoice);
+          this.setState({ waitingWebln: true, openWeblnDialog: true});
+        } else if (data.is_taker & data.status == 3) {
+          webln.sendPayment(data.bond_invoice);
+          this.setState({ waitingWebln: true, openWeblnDialog: true});
+        } else if (data.is_seller & (data.status == 6 || data.status == 7 )) {
+          webln.sendPayment(data.escrow_invoice);
+          this.setState({ waitingWebln: true, openWeblnDialog: true});
+        } else if (data.is_buyer & (data.status == 6 || data.status == 8 )) {
+          webln.makeInvoice(data.amount).then((invoice) => {
```

Comment:
@KoalaSat before merging this PR, take a look at this and the other comments with fixes proposed last night. Otherwise the pipeline for maker/buyers and the submission of the payout invoice is broken.